src/routes/verify/[id]/+page.svelte

```diff
@@ -81,6 +85,28 @@
     </form>
 </div>
 
+<div class="mt-4 flex flex-col items-center justify-center">
+    <div role="alert" class="alert w-3/4">
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            class="stroke-info shrink-0 w-6 h-6"
+            ><path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
+                d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
+            /></svg
+        >
+        <span
+            >{$_('page.verify.spamFolder')}<span class="font-bold text-black"
```

Comment:
This should be one localization string with the value substituted.  Different language has different ways of joining text.  For example, this would not work with Arabic which is right-to-left.

See https://github.com/kaisermann/svelte-i18n/blob/main/docs/Formatting.md#format-_-or-t for how to insert values in a string.